chia/types/mempool_item.py

```diff
@@ -36,6 +35,15 @@ def fee_per_cost(self) -> float:
     def name(self) -> bytes32:
         return self.spend_bundle_name
 
+    @property
+    def cost(self) -> uint64:
+        assert self.npc_result.conds is not None
+        return uint64(self.npc_result.conds.cost)
+
+    @property
+    def additions(self) -> List[Coin]:
+        return additions_for_npc(self.npc_result)
```

Comment:
I wonder if we can include additions (and removals) in `NPCResult` instead of computing on the python side.

Reply:
removals are already included in both `SpendBundle` and `SpendBundleConditions`.
additions are already included in `SpendBundleConditions`, available by iterating each `Spend`

Reply:
Right, but I meant the need for `additions_for_npc` and any similar computation functions on the python side, on `NPCResult`.

Reply:
`additions_for_npc` is cheap, it just loops over the `spends` and appends the `create_coin` lists together.